packages/database/scripts/refreshDatabase.js

```diff
@@ -50,18 +59,23 @@ class RefreshDatabaseScript extends Script {
     }
 
     this.verifyPsql();
-    this.execDbCommand(`DROP DATABASE IF EXISTS ${dbName}`, { user: dbPgUser });
-    this.execDbCommand(`CREATE DATABASE ${dbName} WITH OWNER ${dbUser}`, { user: dbPgUser });
-    this.execDbCommand('CREATE EXTENSION postgis', { db: dbName, user: dbPgUser });
-    this.execDbCommand(`ALTER USER ${dbUser} WITH SUPERUSER`, { user: dbPgUser });
-    this.exec(`psql -U ${dbUser} -d ${dbName} -f "${dumpPath}"`);
-    this.execDbCommand(`ALTER USER ${dbUser} WITH NOSUPERUSER`, { user: dbPgUser });
+    const dbArgs = { user: dbPgUser, port: dbPort, host: dbHost };
+    this.execDbCommand(`DROP DATABASE IF EXISTS ${dbName}`, dbArgs);
+    this.execDbCommand(`CREATE DATABASE ${dbName} WITH OWNER ${dbUser}`, dbArgs);
+    // Loading GIS extension is optional - it's already loaded when using
+    // postgis/postgis Docker container
+    if (this.args.gis) {
+      this.execDbCommand('CREATE EXTENSION postgis', dbArgs);
```

Comment:
IIRC this command does nothing if the extension already exists, so it should be ok to run always.